### DISCUSSION TITLE
Add PHP `else` to Custom Colour Palette

### DISCUSSION THREAD

**Discussion by aljawaid** [aljawaid/ColorManager#26] [Q&A] [answered]
@creecros 
I tried to add an `else` to this block but it didn't work... no errors either.

https://github.com/aljawaid/ColorManager/blob/57e3538e9e6e52a14152160a06b2afe8b0366b82/Template/config/colors.php#L142

I tried:
`<?php if ($customColors == ''): ?> `
`<?php if ($customColors === ''): ?>`
I even tried a simple `else` or the above as `elseif`

Any insight on why?  I tried a few combinations

My ultimate goal is to add: `<p>This palette contains no colors yet. (add color button)</p>`

**Comment by creecros** [comment, marked as answer]
outdated, and should have updated when I changed the model, in this commit: https://github.com/aljawaid/ColorManager/commit/605556edadb9f2556b0bd50e11a9db2b9a8d14b0

 better question to ask now is, is it empty, because its never going to return a string. 
`if (!empty($customColors))`

**Reply by creecros** [reply]
I should have clarified, change the current if statement to that, and then use a simple else for your portion.

**Reply by aljawaid** [reply]
Show if Custom Colour Palette is Empty #27
